storage/wdpost/wdpost_changehandler.go

```diff
@@ -210,6 +211,7 @@ func (p *proveHandler) run() {
 }
 
 func (p *proveHandler) processHeadChange(ctx context.Context, newTS *types.TipSet, di *dline.Info) {
+	ctx = metrics.AddNetworkTag(ctx)
```

Comment:
same problem with the 2 calls here, we're getting a `ctx` passed in from somewhere, we need to hunt that down and add this tag as close to the source as possible